src/video/sdl_v.cpp

```diff
@@ -74,7 +74,8 @@ static void UpdatePalette(bool init = false)
 		pal[i].unused = 0;
 	}
 
-	SDL_SetColors(_sdl_screen, pal, _local_palette.first_dirty, _local_palette.count_dirty);
+	if (BlitterFactory::GetCurrentBlitter()->GetScreenDepth() == 8)
```

Comment:
codestyle - if statements with a single statement as a body need to be on the same line, or enclosed by braces. Probably want to add the braces

+ other places